metricbeat/module/kubernetes/event/_meta/fields.yml

```diff
@@ -24,6 +24,7 @@
     type: keyword
     description: >
       Message recorded for the given event
+    copy_to: message
```

Comment:
Why `copy_to` and not renaming the field?

Can these messages wrap another message from a container?

Reply:
The part I was concerned here is that it's keyword and might be also used in other ways. @exekias will know more here.

Reply:
Ah right. Hence why I thought we should make everything `keyword` (and add mf `message.text`) ;-)

But this works

Reply:
Still think `message` as `text` was a good call ;-)